cypress/e2e/requests/create-new-request-for-Hold-type.cy.js

```diff
@@ -0,0 +1,108 @@
+import { TestTypes, DevTeams, Permissions } from '../../support/dictionary';
+import { ITEM_STATUS_NAMES, REQUEST_TYPES, REQUEST_LEVELS } from '../../support/constants';
+import UserEdit from '../../support/fragments/users/userEdit';
+import TopMenu from '../../support/fragments/topMenu';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import Requests from '../../support/fragments/requests/requests';
+import Users from '../../support/fragments/users/users';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import NewRequest from '../../support/fragments/requests/newRequest';
+import RequestsSearchResultsPane from '../../support/fragments/requests/requestsSearchResultsPane';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import RequestDetail from '../../support/fragments/requests/requestDetail';
+
+describe('Request', () => {
+  let userData = {};
+  let requestId;
+  let itemBarcode;
+  const testData = {
+    folioInstances: InventoryInstances.generateFolioInstances({
+      count: 1,
+      status: ITEM_STATUS_NAMES.CHECKED_OUT,
+    }),
+    servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+  };
+
+  before('Preconditions:', () => {
+    cy.getAdminToken();
+    ServicePoints.createViaApi(testData.servicePoint);
+    testData.defaultLocation = Location.getDefaultLocation(testData.servicePoint.id);
+    Location.createViaApi(testData.defaultLocation).then((location) => {
+      InventoryInstances.createFolioInstancesViaApi({
+        folioInstances: testData.folioInstances,
+        location,
+      });
+    });
+
```

Comment:
according to the test case, you should have checked out item.
please check out item

Reply:
precondition: Any Checked out Item with barcode exists.
![image](https://github.com/folio-org/stripes-testing/assets/100152682/e968ddac-f498-4707-be56-2aa28a189138)
so I've created folioInstance with item wich has ITEM_STATUS_NAMES.CHECKED_OUT.

Could you please clarify what exactly I have to do...